src/main/java/bsise/server/reply/ReplyController.java

```diff
@@ -44,14 +44,15 @@ public List<ReplyResponseDto> getTopNLetters(
         return replyService.findTopNLetterAndReply(size);
     }
 
-    @Operation(summary = "유저의 편지함 목록을 반환하는 API", description = "유저가 최근 작성한 편지와 답변들을 제공합니다.")
+    @Operation(summary = "유저의 편지함 목록을 반환하는 API", description = "연도별 유저가 작성한 편지와 답변들을 최신순으로 제공합니다.")
     @GetMapping(value = "/users/{userId}", produces = MediaType.APPLICATION_JSON_VALUE)
     @ResponseStatus(HttpStatus.OK)
     public Page<ReplyResponseDto> findMyLetters(
             @PathVariable("userId") String userId,
+            @RequestParam int year,
```

Comment:
변수명이 변경되어도 프론트 단에서 변경이 없도록 이름을 지정하면 어떨까요?
```suggestion
            @RequestParam("year") int year,
```

Reply:
오 변수명 변경될 것이라는 생각을 못했군요
감사합니다~